module/VuFind/src/VuFind/RecordDriver/Missing.php

```diff
@@ -103,4 +103,14 @@ public function getTitle()
         $title = parent::getShortTitle();
         return empty($title) ? $this->determineMissingTitle() : $title;
     }
+
+    /**
+     * Get an array of all the formats associated with the record.
+     *
+     * @return array
+     */
+    public function getFormats()
+    {
+        return ['other'];
```

Comment:
As a side effect, I think this is going to cause an untranslated "other" text to appear next to placeholder entries for missing records in lists. Would it be better to use 'Unknown' here to leverage existing translations, or does that not work in other ways?

Reply:
Does make sense, I've changed it to "Unknown"